modules/is/manifests/startserver.pp

```diff
@@ -1,7 +1,13 @@
 class is::startserver inherits is::params {
 
+  exec { 'daemon-reload':
+    command     => "systemctl daemon-reload",
```

Comment:
Shall we reformat the code?

Reply:
Resolved https://github.com/wso2/puppet-is/pull/58/commits/b8ba5cf3de87228146971ff913f98bd16021353b